code/datums/abilities/wraith.dm

```diff
@@ -99,7 +99,9 @@
 	target_anything = 1
 	pointCost = 20
 	cooldown = 45 SECONDS //Starts at 45 seconds and scales upward exponentially
-
+	max_range = 4
+	var/datum/targetable/wraithAbility/absorbCorpse/absorbSuccess = 1 //Determines if wraith succesfully used the ability
```

Comment:
```suggestion
	max_range = 4
	///Determines if wraith successfully used the ability
	var/absorbSuccess = FALSE
```
that's not how variables work in DM. you were setting it to be a type but just having it be `1`.
It should also probably be FALSE by default. Change other usages of this var to be TRUE/FALSE.